deploy/helm_charts/dc_website/templates/config_maps.yaml

```diff
@@ -23,7 +23,8 @@ metadata:
   name: nl-config
   namespace: {{ .Values.namespace.name }}
 data:
-  model.yaml: {{ required "NL embeddings file is required" .Values.nl.embeddings | quote }}
+  embeddings.yaml: {{ required "NL embeddings file is required" .Values.nl.embeddings | quote }}
```

Comment:
@shifucun is this done correctly?

Reply:
Need an empty entry below this line https://github.com/datacommonsorg/website/blob/master/deploy/helm_charts/dc_website/values.yaml#L118

Reply:
Done. PTAL